src/mixins/audio_token_controls.js

```diff
@@ -91,28 +88,48 @@
       touchCornerSize: 64,
 
       cursorStyle: 'pointer',
+      mouseDownHandler: function (eventData, target) {
+        target.controls.playControl.heldDown = true;
+      },
       mouseUpHandler: function (eventData, target) {
         var canvas = target.canvas;
 
+        target.controls.playControl.heldDown = false;
         target.playControlPressed && target.playControlPressed(eventData);
         canvas.requestRenderAll();
       },
-
       render: function (ctx, left, top, styleOverride, fabricObject) {
         // fabricObject.scale is the 'base' scale, scaleX and scaleY are identical, and control the actual drawing scale
         var scale = fabricObject.scaleX;
-        var size = this.cornerSize * scale;
+        var size = fabricObject === fabricObject.canvas.getActiveObject()
+          ? this.cornerSize * scale
+          : this.cornerSize;
         var controlImg;
 
         // draw either the default icons or the ones defined by parent (audio_token)
-        controlImg = fabricObject.isPlaying ?
-          fabricObject.pauseControlImage :
-          fabricObject.playControlImage;
+        if (fabricObject === fabricObject.canvas.getIndicatedObject()) {
+          if (fabricObject.controls.playControl.heldDown) {
+            controlImg = fabricObject.isPlaying
+              ? fabricObject.clickedPauseControlImage
+              : fabricObject.clickedPlayControlImage;
+          } else {
+            controlImg = fabricObject.isPlaying
+              ? fabricObject.hoveredPauseControlImage
+              : fabricObject.hoveredPlayControlImage;
+          }
+        } else if (fabricObject.isGrayScaleEnabled) {
+          controlImg = !fabricObject.isPlaying
+            ? fabricObject.grayScalePlayControlImage
+            : null;
```

Comment:
Is it ever the case that the greyScale control would be playing? (I don't think so....but if so we will see the control disappear.  Which is fine - maybe null is better than just always showing the grayScalePlayControlImage, as it will indicate there is something weird happening.)

Reply:
I think greyScale control never be playing - looks like we use this gray state only for not active objects (now just in work preview). Last word is yours